src/ceph-daemon/ceph-daemon

```diff
@@ -196,7 +196,7 @@ def pathify(p):
     return p
 
 def get_hostname():
-    return socket.gethostname()
+    return socket.gethostname().split('.')[0]
```

Comment:
I'm confused  ... `gethostname()` should already returning this .. ?

The commit message implies you want to split the *container* name (rather than *hostname*) ?

Reply:
not always.. on the teuthology machines where i'm testing, for example,
```
>>> import socket
>>> print socket.gethostname()
smithi060.front.sepia.ceph.com
```

Reply:
i can go either way on this... could argue that this is a misconfiguration of the host and that there are cases where someone *needs* a multicomponent hostname?